content/v2/openapi.yml

```diff
@@ -1178,6 +1179,56 @@ paths:
                     $ref: '#/components/schemas/DomainRenewal'
         '404':
           $ref: '#/components/responses/404'
+  '/{account}/registrar/domains/{domain}/restores':
+    post:
+      description: |-
+        Explicitly restores an domain that can not be renewed anymore, if the registry supports this function.
+
+        Your account must be active for this command to complete successfully. You will be automatically charged the restore fee upon successful restore, so please be careful with this command.
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - $ref: '#/components/parameters/Domain'
+      operationId: domainRestore
+      tags:
+        - registrar restore
+      requestBody:
+        $ref: '#/components/requestBodies/DomainRestore'
+      responses:
+        '201':
+          description: Domain restore created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/DomainRestore'
+        '400':
+          $ref: '#/components/responses/400'
+      summary: Restore a domain
+  '/{account}/registrar/domains/{domain}/restores/{domainrestore}':
+    get:
+      description: Retrieves the details of an existing domain restore.
+      summary: Retrieve a domain restore
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - $ref: '#/components/parameters/Domain'
+        - $ref: '#/components/parameters/DomainRestore'
+      operationId: getDomainRestore
+      tags:
+        - registrar restore
+      responses:
+        '200':
+          description: Domain restore details.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/DomainRestore'
+        '404':
+          $ref: '#/components/responses/404'
```

Comment:
Similarly wouldn't 401 be a valid response code?

Reply:
Added in 16530de7e8d1199fb4e2332bc33a4b3d336f0cb9